helpers/parallel.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+function run_tasks_in_parallel() {
+  if ! command -v parallel >/dev/null; then
+    apt-get update >/dev/null
+    apt-get install --yes parallel >/dev/null
+  fi
+
+  TERM=linux
+  task_type=$1
+  echo "Running ${task_type} in Parallel."
+
+  shift
```

Comment:
Probably a good idea to keep the `shift` right next to assigning `task_type`, so it doesn't get lost